locale/ru_RU/countries.xml

```diff
@@ -35,7 +32,7 @@
 	<country name="Бахрейн" code="BH"/>
 	<country name="Бангладеш" code="BD"/>
 	<country name="Барбадос" code="BB"/>
-	<country name="Беларусь" code="BY"/>
+	<country name="Белоруссия" code="BY"/>
```

Comment:
I can patch this, will do it right now. As for the further question on country names, I tend to agree with @asmecher – even if the Debian list is not perfect, it is something that we can refer to without making every single decision on a name or spelling ourselves. We haven't come with a plan to generate the country list automatically from the Debian package yet.